### ISSUE TITLE
Export of multifile-config from Edge

### ISSUE THREAD

**Issue by o1da** [apigee/apigee-config-maven-plugin#52]
What is the best approach to create multifile-config from existing UI configuration? https://github.com/apigee/apigee-config-maven-plugin/issues/4 suggests apigee-migration-tool but this tool doesn't provide all the stuff and also structure of output is different than the apigee-config-maven-plugin expects. Is there any end-to-end solution?

**Comment by ssvaidyanathan** [comment]
Unfortunately, there is a gap there 
@msadasivam - what do you think ?

**Comment by msadasivam** [comment]
Yes, there is some deviation between the formats but it isn't hard to adjust it once you have it exported. 

An export feature can be added to config plugin itself to create a multi-file config export. Looking to a community contribution if someone sees a need.

**Comment by sarfernando** [comment]
Hi - this would be a useful capability to have - we are in the middle of creating the CICD framework using the multi file config approach and running into the formatting issue.

**Comment by msadasivam** [comment]
@sarfernando, send us a PR :)

In the meantime, if you need to transform the files you already have a library like JSMapr could help.
https://github.com/mdunker/JSMapr
With a little scripting this can be a reusable asset.
